rfc-006-supported-software.md

```diff
@@ -0,0 +1,17 @@
+# Involve the Support Team in exotic software choices
+
+## Summary
+
+The support team must be involved in the decision making where major technology choices increase the burden on the support team.
+
+## Problem
+
+Currently the support team are expected to provide both BAU support, and offer improvements to any platforms that delivery teams ship. Where less understood technologies are introduced in to supported applications, this increases the burden on the support team, both in the hours needed to service the running of the application, and in the on-boarding of support team members.
+
+## Proposal
+
+Delivery teams should be empowered to embrace newer technologies where there is commercial justification.
+
+The support team should expect to not be burdened with supporting unknown platforms without having input in to the decision making.
+
+The support team publishes a list of software and platforms (to the granularity of "Python", "Elasticsearch", or "Cloud Foundry") that they currently have capability to support. Where teams wish to deviate from this, and where we need to provide ongoing support, teams MUST involve the support team in the decision making.
```

Comment:
Practically speaking, how would the "we want to use technologies not on that list" scenario play out? It's not going to be a small list and deviating from it would have to be a considered risk from the PoV of the support team. I'm worried those conversations could turn into peer pressure moments where the delivery team say "yeah don't worry, it'll be fine" or the support team don't want to be seen to be getting in the way of evolution, and then hilarity ensues.

Reply:
I have this concern as well, but was having trouble explaining - nice work Scott.

The values of "empower the delivery team" and "consult outside of the delivery team" are hard to reconcile.

Reply:
@kjdchapman I don't see the conflict myself between those two things. The Delivery Team are empowered to choose the best technology to meet the commercial need, and as part of being empowered, there's a responsibility to consult with those who will be affected by the decision. For me, empowered doesn't need to mean in isolation.

@sowasred2012 any suggestion on how to foster healthy conflict between the teams in ratifying these decisions, rather than one team rolling over?

Reply:
@chrisblackburn something along the lines of:

**Risk/benefit assessment:** 
- The delivery team wants to use Shiny New Technology A because Benefit X, Y and Z, so they present their case to the support team, either in a meeting or by some other means. 
- Support team (or representative) asks questions to gain any insight they feel they need to make an assessment. 
- Support team go away and draw up a risk assessment, balance the risks against the benefits, then meet with the delivery team to present their assessment, which I'd be inclined to say should be what makes the decision ultimately.

Introduces a fair bit of red tape, but would give the support team space to investigate the new technology for themselves (rather than relying on the word of a team who have a bias towards it), and consider what impact it would have on their current capabilities.

Reply:
@sowasred2012 how's that most recent edit? I don't necessarily want to mandate *how* you supporters choose to input in to the decision making process, more that where we need to support a technology stack, you need to be OK with it.